changelog.d/12533.doc

```diff
@@ -0,0 +1 @@
+Remove extraneous comma in users_admin_api device deletion section so example JSON is actually valid and working. Contributed by @olmari.
```

Comment:
```suggestion
Remove extraneous comma in User Admin API's device deletion section so that the example JSON is actually valid and works. Contributed by @olmari.
```